templates/org/home.tmpl

```diff
@@ -28,6 +28,9 @@
 			<div class="ui eleven wide column">
 				{{if .CanCreateOrgRepo}}
 					<div class="text right">
+            {{if not .DisabledMirrors}}
+						  <a class="ui green button" href="{{AppSubUrl}}/repo/migrate?org={{.Org.ID}}&mirror=1">{{.i18n.Tr "new_mirror"}}</a>
```

Comment:
```suggestion
						  <a class="ui green button" href="{{AppSubUrl}}/repo/migrate?org={{.Org.ID}}&mirror=1">{{.i18n.Tr "new_migrate"}}</a>
```
Matches the text in the "+" Button on the header.

Reply:
The poster from #13091 specifically mentions a new mirror button. Currently, as you can see in screenshot 3, the mirror option will be ticked. I assumed that is what the user wanted. @somera can you confirm?

Reply:
It's called "Migration" in the top menu and it links to a page for migrations so I'd say the button should be named consistently like that.